apis/v1beta1/object_reference_types.go

```diff
@@ -145,42 +145,3 @@ type BackendObjectReference struct {
 	// +optional
 	Port *PortNumber `json:"port,omitempty"`
 }
-
-// ObjectReference identifies an API object including its namespace, with no
```

Comment:
Yeah, I guess taking this out for now is fine - I originally left it in because I think we will need to swap to it eventually, but  moving from LocalObjectReference to this is not too breaking later.